lib/ap_configurator/ap_configurator.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/python3
```

Comment:
This is not the correct way to specify a python shebang and will make trouble on the phone on termux. Use !/usr/bin/env python